index.css

```diff
@@ -0,0 +1,29 @@
+body {
+    background-color: lavender;
+    font-family: monospace;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+main {
+    width: 60%;
+    display: flex;
+    justify-content: left;
+    flex-direction: column;
+}
+
+img {
+    height: 150px;
+}
+
+header {
```

Comment:
To align the text in the middle via y-axis you could of used the align-items:center property